terraform/groups/ecs-service/locals.tf

```diff
@@ -3,18 +3,23 @@ locals {
   stack_name                  = "company-requests" # this must match the stack name the service deploys into
   name_prefix                 = "${local.stack_name}-${var.environment}"
   global_prefix               = "global-${var.environment}"
-  service_name                = "stike-off-objections-api"
+  service_name                = "strike-off-objections-api"
   container_port              = "8080"
   eric_port                   = "10000"
-  docker_repo                 = "stike-off-objections-api"
+  docker_repo                 = "strike-off-objections-api"
   kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority   = 31
+<<<<<<< Updated upstream
   lb_listener_paths           = ["/company/*/strike-off-objections*"]
   healthcheck_path            = "/strike-off-objections-api/healthcheck" #healthcheck path for stike-off-objections-api
+=======
+  lb_listener_paths           = ["/strike-off-objections-api-reports*"]
+  healthcheck_path            = "/strike-off-objections-api-reports/healthcheck" #healthcheck path for strike-off-objections-api
+>>>>>>> Stashed changes
```

Comment:
This needs an update for the merge conflicts. The lb_listener_paths need to match the path mappings used by the application